src/AppBundle/Repository/AutoEditsRepository.php

```diff
@@ -61,7 +90,7 @@ public function getTools(Project $project, $namespace = 'all'): array
     public function countAutomatedEdits(Project $project, User $user, $namespace = 'all', $start = '', $end = ''): int
     {
         $cacheKey = $this->getCacheKey(func_get_args(), 'user_autoeditcount');
-        if ($this->cache->hasItem($cacheKey)) {
+        if ($this->cache->hasItem($cacheKey) && !$this->useSandbox) {
```

Comment:
here and below, suggest putting the useSandbox check first to avoid checking the cache unnecessarily, and to be consistent with the order in AutomatedEditsHelper.php (or, if checking second is intentional, suggest being consistent with that)

Reply:
In production it makes virtually no difference, since it's in-memory APCu cache, but good practice nonetheless!